src/odata.cr

```diff
@@ -0,0 +1,18 @@
+module Office365::OData
+  extend self
+
+  # (key1 in ('value1', 'value2', ...))
+  def in(key : String, values : Array(_)) : String
+    wrap(key, "in", array(values))
+  end
+
+  # ('value1', 'value2', ...)
+  def array(values : Array(_)) : String
+    wrap(values.map { |v| "'#{v}'" }.join(", "))
+  end
+
+  # ('value1', 'value2', ...)
+  def wrap(*args) : String
+    "(" + args.map(&.to_s).join(" ") + ")"
```

Comment:
Can use the join with block method outlined in other comments here.
I would also use a template string rather than concatenation to prevent multiple string allocations.